runtime/doc/options.txt

```diff
@@ -2114,6 +2114,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	each enabling fuzzy collection for a specific completion mode:
 	files		file names
 	keyword		keyword completion in 'complete' and current file
+			Note only regular keyword can be fuzzy collected.
```

Comment:
Does that mean only for `i_CTRL-N/P` but not for `i_CTRL-X_CTRL-N/P`? Then perhaps add a link to the keyword completion method that is supported.
Also, I stumbled over `keyword completion in 'complete' and current file` already last time, and cannot make sense of it. Can you please clarify this?

Reply:
CTRL_X _CTRL_N/P also supported the test case use them ..

Reply:
documentation has been updated to note that it is not that other modes will not be supported. We can add them one by one, so that we don't add a lot of code at once, and any problems will be easier to fix.

Reply:
thanks, looks much better now.